FlashSpace/Settings/WorkspacesSettingsView.swift

```diff
@@ -30,6 +31,33 @@ struct WorkspacesSettingsView: View {
                 .foregroundStyle(.secondary)
                 .font(.callout)
             }
+
+            Section(
+                header:
+                HStack {
+                    Text("Floating Apps")
+                    Spacer()
+                    Button(action: viewModel.addFloatingApp) {
+                        Image(systemName: "plus")
+                    }
+
+                    Button(action: viewModel.deleteFloatingApp) {
+                        Image(systemName: "trash")
+                    }.disabled(viewModel.selectedFloatingApp == nil)
+                }
+            ) {
+                VStack(alignment: .leading) {
+                    List(
+                        settings.floatingApps ?? [],
+                        id: \.self,
+                        selection: $viewModel.selectedFloatingApp
+                    ) {
+                        Text($0)
+                    }
+                }
+                hotkey("Float The Focused App", for: $settings.floatTheFocusedApp)
+                hotkey("Unfloat The Focused App", for: $settings.unfloatTheFocusedApp)
+            }
```

Comment:
I think it would be nice to have 
![x circle fill@2x](https://github.com/user-attachments/assets/0cf155fd-17e6-4e59-84ea-ba8678770997)
`x.circle.fill`

next to each entry to quickly delete apps, instead of each time two clicks - select, delete, select, delete. What do you think?

Maybe even the same could be done for apps on the MainView (but leave it for later).

Reply:
Okay, I fixed it as you said.